deps/openssl/openssl/apps/app_rand.c

```diff
@@ -124,10 +124,16 @@ int app_RAND_load_file(const char *file, BIO *bio_e, int dont_warn)
     char buffer[200];
 
 #ifdef OPENSSL_SYS_WINDOWS
-    BIO_printf(bio_e, "Loading 'screen' into random state -");
-    BIO_flush(bio_e);
-    RAND_screen();
-    BIO_printf(bio_e, " done\n");
+    /*
+     * allocate 2 to dont_warn not to use RAND_screen() via
+     * -no-rand-screen option in s_client
+     */
+    if (dont_warn != 2) {
```

Comment:
`dont_warn` 2 is only for the use this fix and more than 2 are not defined. I think it would rather be better to let the only 2 allocate to skip `RAND_screen()`.